app/scripts/filters/resources.js

```diff
@@ -125,6 +125,50 @@ angular.module('openshiftConsole')
       return displayName;
     };
   })
+  .filter('canI', function(AuthorizationService) {
+    return function(resource, verb) {
+      return AuthorizationService.canI(resource, verb);
+    };
+  })
+  .filter('canIDoAny', function(canIFilter) {
+    var resourceRulesMap = {
```

Comment:
@liggitt plural is for pages in which we list the resource, singular for browse page.

@jwforres are you talking about other-resource rules that are listed here, or generally ? Problem is that dropdown often contains action on more then one resource, eg `buildConfig`, where we have to check `buildConfig` for `delete` and `update` action and `buildconfigs/instantiate` for `create` action.  But we can't pass to the filter two arrays {[resources], [verbs]} cause we want to check specific action on specific resource, so it has to be a collection, like the one I've proposed.
Also I've didn't come up with a better place to put the collection since it can't go to a view, or at least it would look ugly:
`ng-hide="!({'buildconfigs': ['delete', 'update'], 'buildconfigs/instantiate': ['create']} | canIDoAny)"`

What we could do is do leave the other-resources out of this, and just use the canI filter to check the specific resource for update and delete action, since we only display those two.